YIF_Backend/Controllers/InstitutionOfEducationController.cs

```diff
@@ -211,5 +211,22 @@ public async Task<IActionResult> DeleteInstitutionOfEducationFromFavorite(string
             var result = await _institutionOfEducationService.GetInstitutionOfEducationAbbreviations(filterModel);
             return Ok(result);
         }
+
+        /// <summary>
+        /// Get all directions and specialties of institution of education
+        /// </summary>
+        /// <response code="200">Get all directions and specialties in institution of education</response>
+        /// <response code="400">If there is no user with such id</response>
```

Comment:
isn't it code 404 not found?

Reply:
This is correct.
404 is used, when url not found.
I take user id from context and throw 400 bad request, if id is not found.